sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -579,6 +579,155 @@ class AdaptiveQueryExecSuite
     }
   }
 
+  test("SPARK-29544: adaptive skew join both in left and right for inner join," +
```

Comment:
nit: `adaptive skew join with different join types`, to make the test name shorter.